src/components/Tabs/Tabs.stories.mdx

```diff
@@ -108,6 +123,40 @@ export const animalCrossing = [
 
 <ArgsTable story="Tabs with Controls" />
 
+## Accessibility
+
+Using the DS `Tabs` component renders HTML for button tabs and panels that are
+associated with each other. In order to simplify the content and reduce cognitive
+load for a user, we recommend to not use more than a combination of six (6) button
+tabs and six (6) panels. Likewise, button tab labels should be clear and concise.
```

Comment:
This confused me a bit because you cannot have more panels than buttons or the other way around. I think you could say "...a combination of six (6) button tabs and panels." Unless I am misunderstanding.